controls/cis_al2023.yml

```diff
@@ -0,0 +1,2176 @@
+    rules:
+      - set_password_hashing_algorithm_systemauth
+      - set_password_hashing_algorithm_passwordauth
+      - set_password_hashing_algorithm_logindefs
+      - var_password_hashing_algorithm=SHA512
+
+  - id: 4.6.1.1
+    title: Ensure password expiration is 365 days or less (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_maximum_age_login_defs
+      - var_accounts_maximum_age_login_defs=365
+      - accounts_password_set_max_life_existing
+
+  - id: 4.6.1.2
+    title: Ensure minimum days between password changes is configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_minimum_age_login_defs
+      - var_accounts_minimum_age_login_defs=1
+      - accounts_password_set_min_life_existing
+
+  - id: 4.6.1.3
+    title: Ensure password expiration warning days is 7 or more (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_password_warn_age_login_defs
+      - var_accounts_password_warn_age_login_defs=7
+      - accounts_password_set_warn_age_existing
+
+  - id: 4.6.1.4
+    title: Ensure inactive password lock is 30 days or less (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - account_disable_post_pw_expiration
+      - var_account_disable_post_pw_expiration=30
+      - accounts_set_post_pw_existing
+
+  - id: 4.6.1.5
+    title: Ensure all users last password change date is in the past (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_password_last_change_is_in_past
+
+  - id: 4.6.2
+    title: Ensure system accounts are secured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - no_password_auth_for_systemaccounts
+      - no_shelllogin_for_systemaccounts
+
+  - id: 4.6.3
+    title: Ensure default user shell timeout is 900 seconds or less (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_tmout
+      - var_accounts_tmout=15_min
+
+  - id: 4.6.4
+    title: Ensure default group for the root account is GID 0 (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_root_gid_zero
+
+  - id: 4.6.5
+    title: Ensure default user umask is 027 or more restrictive (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_umask_etc_bashrc
+      - accounts_umask_etc_login_defs
+      - accounts_umask_etc_profile
+      - var_accounts_user_umask=027
+
+  - id: 4.6.6
+    title: Ensure root password is set (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - ensure_root_password_configured
+      - no_empty_passwords_etc_shadow
+
+  - id: 5.1.1.1
+    title: Ensure rsyslog is installed (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - package_rsyslog_installed
+
+  - id: 5.1.1.2
```

Comment:
This could probably be automated as well. Especially since you defined "rules".

Reply:
You can either change to "automated" or let it "manual" and replace `rules` by `related_rules`.